### DISCUSSION TITLE
Tezos Support??

### DISCUSSION THREAD

**Discussion by murkai** [nftchef/art-engine#18] [Ideas]
Hi Chef! Wondering, in case anyone else is interested, can we please get Tezos support in your fork? 

Another Dev came up with this: https://github.com/tqtezos/minter

but it's missing all the awesome sauce of your fork (nested structure, omitting metadata during generation, etc) 

Thanks in advance :)

**Comment by nftchef** [comment]
Thanks for the request.
It appears that tezos would be able to be supported by outputting metadata in the format outlined here

https://tzip.tezosagora.org/proposal/tzip-21/

I will do a little investigating and work on it if possible.

**Reply by murkai** [reply]
Thanks so much @nftchef for replying so quickly! This would be an incredible add! Please keep us posted :)

**Comment by nftchef** [comment]
https://github.com/vivekascoder/nftchef-art-engine-tz

**Comment by nftchef** [comment]
Tezos supported added in #111
